data/mods/gen5/random-teams.ts

```diff
@@ -213,8 +226,12 @@ export class RandomGen5Teams extends RandomGen6Teams {
 			return {cull: moves.has('bodyslam')};
 		case 'healbell':
 			return {cull: !!counter.get('speedsetup') || moves.has('magiccoat')};
-		case 'moonlight': case 'painsplit': case 'recover': case 'roost': case 'softboiled': case 'synthesis':
+		case 'moonlight': case 'painsplit': case 'recover': case 'softboiled': case 'synthesis':
 			return {cull: ['leechseed', 'rest', 'wish'].some(m => moves.has(m))};
+		case 'roost':
```

Comment:
It's unnecessary to separate this; the gliscorCase clause works just fine alongside the other recovery moves and doesn't mess anything up

Reply:
You're right, thanks!